### PR TITLE
Check My POSt in React completed

### PR DESCRIPTION
**Pull request by christinem**
This completes the Check My POSt in React, including removing all the old Javascript files.

### CODE REVIEW

File: public/js/post/post_components.js [hunk 1]
```diff
@@ -114,6 +136,23 @@ var Post = React.createClass({
...
+            completed: false
+        }
+    },
+
+    changeTabView: function(isSelected) {
+        this.setState({selected: isSelected});
+    }, 
+
+    setIfCompleted: function() {
+        var isCompleted = this.refs.post.state.creditCount >= 12.0;
```
Comment (david-yz-liu) [inline public/js/post/post_components.js]: Make the credit number a prop

File: public/js/post/post_page.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    updateNavCreditCounts: function() {
+        var newCounts = [this.refs.spePost.getCreditCount(), 
+                         this.refs.majPost.getCreditCount(),
+                         this.refs.minPost.getCreditCount()];
+        this.refs.postNav.setState({creditCounts: newCounts},
+            this.updatePostStatus());
+    },
+
+    updatePostStatus: function() {
+
```
Comment (david-yz-liu) [inline public/js/post/post_page.js]: Delete this line

File: public/js/post/post_page.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+var PostNav = React.createClass({
+    getInitialState: function() {
+        return {
+            visible: this.getActiveTab() === '' ? 'spe' : this.getActiveTab(),
+            creditCounts: [0.0, 0.0, 0.0],
+            completed: [false, false, false]
+        }
+    },
+
+    getActiveTab: function() {
```
Comment (david-yz-liu) [inline public/js/post/post_page.js]: Remove this function; save the value in a temporary variable in `getInitialState`.

**Comment by david-yz-liu** [comment]
@christinem nice job. I left one minor comment for you.

I noticed it's pretty slow when switching between POSts. Looks like a bunch of server requests are being sent for the different courses for "getTitle" - this really shouldn't happen every time. You could look into setting the course title as state on the initial mounting.

**Comment by christinem** [comment]
@david-yz-liu thanks! I didn't realize that what was making it slower - I'll take a look at that (along with the other comments).

**Pull request merged**